src/agri_ai/langchain_tools/field_registration_tool.py

```
@@ -0,0 +1,252 @@
+
+    def _format_result(self, result: Dict[str, Any]) -> str:
+        """結果のフォーマット"""
+        if result.get("error"):
+            return f"❌ {result['error']}"
+        
+        if result.get("success"):
+            field_info = result["field_info"]
+            formatted_lines = [
+                "✅ 圃場登録が完了しました！",
+                "",
+                f"📋 圃場情報",
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)